source/adminguide/virtual_machines.rst

```diff
@@ -960,6 +960,115 @@ restoreVirtualMachine call. In this case, the Instance's root disk is
 destroyed and recreated, but from the same Template or ISO that was
 already in use by the Instance.
 
+Instance Lease
+--------------
+
+Cloudstack provides capability to create instance on lease. Lease denotes a set period for which resource is allocated and upon expiry cleanup is performed.
+This feature enables automated cleanup of instances created for specific duration and for specific purpose. This feature gives administrators the ability to automatically reclaim
+resources that are no longer needed by expired virtual machines, helping to optimize resource utilization and reduce wastage.
+
+
+**Configuring lease feature**
+
+The cloud administrator can use global configuration variables to control the behavior of Instance Lease.
+To set these variables, API or CloudStack UI can be used:
+
+.. cssclass:: table-striped table-bordered table-hover
+
+======================================= ========================
+Configuration                            Description
+======================================= ========================
+instance.lease.enabled                   Indicates whether to enable the Instance lease featuew, will be applicable only on instances created after lease is enabled. **Default: false**
+instance.lease.scheduler.interval		  Background task interval in seconds that executes Lease expiry action on eligibile expired instances. Default: 3600.
+instance.lease.alertscheduler.interval	  Background task interval in seconds that executes Lease alert for instances about to be expired in next N days. Default: 86400
+instance.lease.alert.daysbefore          Denotes number of days (N) for alert task. Default: 7 days
+======================================= ========================
+
+
+**Using Instance Lease**
+
+User can associate Lease to an instance during Deployment of instance or modify existing lease by editing instance.
```

Comment:
Worth mentioning, the behaviour around what happens:
* When instances are not deployed with leases, can they acquire lease by switching service offering?
* What is allowed for admins & end-users for instances deployed with lease
* Generally when service offering is changed
* When lease is disabled by user or admin for an instance